paddle/majel/place.cc

```diff
@@ -1,10 +1,11 @@
-#include <majel/place.h>
+#include "majel/place.h"
+#include "mapbox/variant.hpp"
 
 namespace majel {
 
 namespace detail {
 
-class PlacePrinter : public boost::static_visitor<> {
+class PlacePrinter : public mapbox::util::static_visitor<> {
```

Comment:
What is the difference between `mapbox::util::static_visitor` and `std::static_visitor`?  Should we use the latter?